zerver/tests/test_event_system.py

```diff
@@ -1245,7 +1245,7 @@ def test_queries(self) -> None:
             realm_user=4,
             realm_user_groups=3,
             realm_user_settings_defaults=1,
-            recent_private_conversations=1,
+            recent_private_conversations=2,
```

Comment:
What is the new query this adds? I don't see a reason this should add a query at all to fetching recent private conversations...